src/Riok.Mapperly/Helpers/EnumNamingStrategyHelper.cs

```diff
@@ -0,0 +1,59 @@
+using System.Globalization;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis;
+using Riok.Mapperly.Abstractions;
+
+namespace Riok.Mapperly.Helpers;
+
+public static class EnumNamingStrategyHelper
+{
+    public static Dictionary<IFieldSymbol, string> BuildCustomNameStrategyMappings(
+        this ITypeSymbol enumSymbol,
+        EnumNamingStrategy namingStrategy
+    )
+    {
+        var customNameMappings = new Dictionary<IFieldSymbol, string>(SymbolEqualityComparer.Default);
+
+        var values = enumSymbol.GetMembers().OfType<IFieldSymbol>();
```

Comment:
It would probably make sense to use the symbol accessor to access the members.

Reply:
Yes, agreed. I have changed it to use the symbol accessor to access the members.